include/ppx/knob.h

```diff
@@ -275,7 +303,12 @@ class KnobDropdown final
     // Needed for setting from flags but use is discouraged otherwise
     void SetIndex(const std::string& newValue)
```

Comment:
Should this be `SetIndexByValue` and `SetIndexByName` to provide flexibility? Although if there is no use for that, we can always add that later.

Reply:
Nothing uses it, so deleted instead, we can add `SetIndexBy{Name,Value}` when we needs them.